lib/server/utils.js

```diff
@@ -29,6 +30,7 @@ function getLanguage(file, refDir) {
   const match = regexSubFolder.exec(file);
 
   // Avoid misinterpreting subdirectory as language
+  const env = require('./env.js');
```

Comment:
Good idea here 😂But I think we'll need to change this back to top level when we move to ES6 as ES6 imports have to be top level.